server/src/main/java/org/opensearch/action/admin/indices/upgrade/post/ShardUpgradeRequest.java

```diff
@@ -1,65 +0,0 @@
-/*
```

Comment:
@nknize do you think we could move these changes to separate pull request (as you did with aliases & types)?

Reply:
Sure thing!